stanford_profile.profile

```diff
@@ -214,6 +214,11 @@ function stanford_profile_config_pages_stanford_basic_site_settings_form_validat
     if (!$match) {
       $form_state->setErrorByName('su_site_url', t('Only valid stanford.edu domain names allowed.'));
     }
+    $host = \Drupal::request()->getSchemeAndHttpHost();
+    if ($host != $uri) {
+      $form_state->setErrorByName('su_site_url', t('Your current domain does not match the supplied domain.'));
+    }
```

Comment:
So this would force the user to be logged in on the URL they want to set the site to? I think that makes sense. @pookmish @jbickar Thoughts on this? Would this validation break the drush commands/scripts we use to set it?

@boznik can you update the help text to indicate this new restriction.

Reply:
Its a bit more restrictive than using the service from the domain_301_redirect module but it would accomplish similar validation and would be faster than the service. 
And it won't affect drush commands. we don't have any current commands to set the value, only to clear it from the database.

Reply:
I did try using the service from the domain_301_redirect module, but it returned false on the checkDomain method.

Reply:
That service will only work on live publicly accessible urls. It works similarly to lets encrypt verification.

Reply:
gotcha

Reply:
Thanks for the feedback. @boznik update the help text on the field and then this should be GTG.

Reply:
It would be nice to have a drush command to set it, but yes, we don't have one currently.

Reply:
theres supposed to be one, but i couldn't get it to work. maybe i just need to do it right. :shrug: `drush config:pages-set-field-value`